content/docs/concepts/security/index.md

```diff
@@ -191,7 +191,7 @@ real-world scenario.
   that both teams can't access each other's services.
 
 * If Citadel is compromised, all its managed keys and certificates in the
```

Comment:
this should be a warning.  Please examine https://preliminary.istio.io/about/contribute/writing-a-new-topic/#adding-icons-emojis.